server/src/main/java/com/objectcomputing/checkins/services/skills/Skill.java

```diff
@@ -74,6 +88,22 @@ public void setPending(boolean pending) {
         this.pending = pending;
     }
 
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public boolean isExtraneous() {
+        return extraneous;
+    }
+
+    public void setExtraneous(boolean extraneous) {
+        this.extraneous = extraneous;
+    }
+
     @Override
     public boolean equals(Object o) {
```

Comment:
Should we add `description` and `extraneous` fields to `equals`, `hashcode` and `toString` methods?

Reply:
Of course, good catch!  I will make the changes.